src/Css/CodeMap.module.css

```diff
@@ -0,0 +1,217 @@
+.App {
+    font-family: sans-serif;
+    text-align: center;
+  }
+  
+  .btn {
+    background-color: #008dc8;
+    float: right;
+    font: Regular 30px/40px Segoe UI;
```

Comment:
@Kritimukhija This is some Google font but I don't see it being imported anywhere?

CC: @Manvityagi